ddtrace/contrib/requests/connection.py

```diff
@@ -97,13 +97,11 @@ def _wrap_send(func, instance, args, kwargs):
             # Storing response headers in the span. Note that response.headers is not a dict, but an iterable
             # requests custom structure, that we convert to a dict
             if hasattr(response, "headers"):
-                store_response_headers(dict(response.headers), span, config.requests)
+                trace_utils.set_http_meta(span, config.requests, headers=dict(response.headers))
```

Comment:
These are response headers and not request headers so we probably don't want to set them here.